### ISSUE TITLE
GROW -Ingest JGI sequencing and analysis data records for metatranscriptomes

### ISSUE THREAD

**Issue by aclum** [microbiomedata/issues#58]
There are 14 metatranscriptome projects associated with Gs0149396. This is lower priority for GSP than processing the metagenomes b/c there are more of them.
Depends on [Add missing file_name_pattern to file type enums for metatranscriptomes#708](https://github.com/microbiomedata/nmdc-schema/issues/708) - resolved

blocked on https://github.com/microbiomedata/nmdc_automation/issues/227

**Comment by aclum** [comment]
Not a target for GSP so updating labels.

**Comment by ssarrafan** [comment]
Does this still need to be done @aclum 
@scanon @mflynn-lanl are you working on this one?

**Comment by aclum** [comment]
No one is working on this but this is still needed. Once https://github.com/microbiomedata/nmdc_automation/pull/223 is approved we'll need a matching one for JGI ingest. Likely won't happen until after rolling out the berkeley schema.

**Comment by ssarrafan** [comment]
Berkeley schema rolled out long time ago.. is this still on your radar? I'm removing from post GSP board. @aclum

**Comment by aclum** [comment]
yes, we still need to do this, it is on the dataset priority board.

**Comment by aclum** [comment]
This was completed this summer.
